Add details about rate limits for Operator deployments (#2748)
wandb/docs: platform/hosting/self-managed/operator.mdx

    GLOBAL_ENV: "example"
```

### Set rate limits

For Dedicated Cloud and Self-Managed deployments that use the Operator, optionally configure rate limits by setting environment variables in `spec.values.global.extraEnv`. This example shows the default value when the variable is not set.

<Important>You must set `GORILLA_LIMITER` to the address of your Redis server.</Important>

```yaml
spec:
  values:
    global:
      extraEnv:
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_COUNT: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_PER_RUN_COUNT: "0.8"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_SIZE: "10"
        GORILLA_DEFAULT_RATE_LIMITS_GRAPHQL: "50"
        GORILLA_DEFAULT_RATE_LIMITS_RUN_UPDATE_COUNT: "10"

>         GORILLA_DEFAULT_RATE_LIMITS_SDK_GRAPHQL_QUERY_SECONDS: "1"

Reviewer: I would maybe remove the `GORILLA_DEFAULT_RATE_LIMITS_SDK_GRAPHQL_QUERY_SECONDS` and `GORILLA_DEFAULT_RATE_LIMITS_GRAPHQL` from the recommendation since we don't set these on dedicated by default.

>         GORILLA_LIMITER: "redis://redis-service:6379?ttlInSeconds=900"

Author: ```suggestion
        GORILLA_LIMITER: "redis://[redis-service-address]:6379?ttlInSeconds=900"
```

<Important>To enable rate limiting, you must set `GORILLA_LIMITER` to the address of your Redis server.</Important>

```yaml
spec:
  values:
    global:
      extraEnv:
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_COUNT: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_PER_RUN_COUNT: "0.8"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_SIZE: "10"
        GORILLA_DEFAULT_RATE_LIMITS_RUN_UPDATE_COUNT: "10"
        GORILLA_LIMITER: "redis://$HOST:6379?ttlInSeconds=900"
```

Refer to the following table for more detail:

| Environment variable | Default | Description |
| --- | --- | --- |
| `GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM` | `5` | Default filestream request limit. |
| `GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_COUNT` | `5` | Filestream requests per second. |
| `GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_PER_RUN_COUNT` | `0.8` | Filestream requests per run per second. |
| `GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_SIZE` | `10` | Filestream ingestion limit in MB per second. |

> | `GORILLA_DEFAULT_RATE_LIMITS_GRAPHQL` | `50` | GraphQL request limit. |

Author: ```suggestion
```

| `GORILLA_DEFAULT_RATE_LIMITS_RUN_UPDATE_COUNT` | `10` | Run metadata update requests per second. |

> | `GORILLA_DEFAULT_RATE_LIMITS_SDK_GRAPHQL_QUERY_SECONDS` | `1` | SDK GraphQL query seconds budget. |

Author: ```suggestion
```

    GLOBAL_ENV: "example"
```

### Set rate limits

> For Dedicated Cloud and Self-Managed deployments that use the Operator, optionally configure rate limits by setting environment variables in `spec.values.global.extraEnv`. This example shows the default value when the variable is not set.

Author: ```suggestion
For Dedicated Cloud and Self-Managed deployments that use the Operator, optionally configure rate limits by setting environment variables in `spec.values.global.extraEnv`. For reference, this example explicitly sets each rate limit to its default. In practice, define a rate limit only to override the default.
```

> <Important>You must set `GORILLA_LIMITER` to the address of your Redis server.</Important>

Author: ```suggestion
<Important>To enable rate limiting, you must set `GORILLA_LIMITER` to the address of your Redis server.</Important>
```

    GLOBAL_ENV: "example"
```

### Set rate limits

For Dedicated Cloud and Self-Managed deployments that use the Operator, optionally configure rate limits by setting environment variables in `spec.values.global.extraEnv`. This example shows the default value when the variable is not set.

<Important>You must set `GORILLA_LIMITER` to the address of your Redis server.</Important>

```yaml
spec:
  values:
    global:
      extraEnv:
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_COUNT: "5"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_PER_RUN_COUNT: "0.8"
        GORILLA_DEFAULT_RATE_LIMITS_FILESTREAM_SIZE: "10"

>         GORILLA_DEFAULT_RATE_LIMITS_GRAPHQL: "50"

Author: ```suggestion
```

        GORILLA_DEFAULT_RATE_LIMITS_RUN_UPDATE_COUNT: "10"

>         GORILLA_DEFAULT_RATE_LIMITS_SDK_GRAPHQL_QUERY_SECONDS: "1"

Author: ```suggestion
```
